jhipster-framework/src/main/java/io/github/jhipster/config/metric/JHipsterMetricsEndpoint.java

```diff
@@ -0,0 +1,272 @@
+package io.github.jhipster.config.metric;
```

Comment:
Try to harmonize the code style of this big class.
- Consistent spacing
- Use camel case guidelines for acronyms , eg. resultsHTTPAll -> resultsHttpAll (I know this is a big debate in java 😄 https://stackoverflow.com/a/7366332/6896299)